bin/fuel-core/src/cli/rollback.rs

```diff
@@ -0,0 +1,93 @@
+use crate::cli::default_db_path;
+use anyhow::Context;
+use clap::Parser;
+use fuel_core::{
+    combined_database::CombinedDatabase,
+    service::genesis::NotifyCancel,
+    state::historical_rocksdb::StateRewindPolicy,
+};
+use std::path::PathBuf;
+
+/// Print a snapshot of blockchain state to stdout.
```

Comment:
Is this rustdoc copied and pasted from some other command? It seems like this rewinds the database and doesn't print anything at all to std-out.